opengrok-tools/src/test/python/test_commandSequence.py

```diff
@@ -77,6 +77,20 @@ def test_exit_2_handling():
     assert not cmds.failed
 
 
+@pytest.mark.skipif(not os.path.exists('/bin/sh') or not os.path.exists('/bin/echo'), reason="requires Unix")
+def test_driveon_flag():
+    cmds = CommandSequence(CommandSequenceBase("opengrok-master",
+                                               [{"command": ["/bin/sh", "-c",
+                                                             "echo " + CommandSequence.PROJECT_SUBST + "; exit 2"]},
```

Comment:
is this 80 columns?

Reply:
most likely not. the test files are still exempted from style checks I think.

Reply:
The reason was originally path manipulation however this is gone so probably we can remove the excludes.

Reply:
yes